packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -219,6 +219,15 @@ library LibProving {
                 revert L1_INVALID_EVIDENCE();
             }
 
+            if (
+                evidence.proof.length != 0
```

Comment:
```suggestion
                evidence.proof.length == 1
```

Reply:
I think != 0 is easier because then in the GuardianVerifier.sol it can stay as is - if for whatever reason the guardian is not returning the liveness bond. What do you think ?

Reply:
In GuardianVerifier.sol we can simply remove "if (evidence.proof.length != 0) revert INVALID_PROOF();"